.github/actions/build-ab/index.js

```diff
@@ -133,3 +134,68 @@ if (['dev', 'staging'].includes(args.environment)) {
     }
   }
 }
+
+// add entitlements NG script shim
+(async () => {
+  if (environment === 'dev') return
+  const entitlementsEndpoints = {
+    dev: '',
+    staging: 'https://staging-one.newrelic.com',
+    prod: 'https://one.newrelic.com',
+    'eu-prod': 'https://one.eu.newrelic.com'
```

Comment:
Are these not the same domains for the UI? Couldn't we just use a relative URL in the XHR?

Reply:
yeah i guess thats true.  we can go to a relative url.  Theres the off chance that the API and the UI segregate at some point but i think that we can cross that bridge if and when that ever happens.